### PR TITLE
Bala/capture cf pages preview URL to run lighthouse and smoke test

### PR DESCRIPTION
**Pull request by balakrishna-deriv**
Changes:

-   Capture cf pages preview URL to run lighthouse and smoke test

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: .github/workflows/lighthouse.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  contents: write
-  deployments: write
-  pull-requests: write
-  statuses: write
+    actions: write
+    checks: write
+    contents: write
+    deployments: write
+    pull-requests: write
+    statuses: write
```
Comment (balakrishna-deriv) [inline .github/workflows/lighthouse.yml:9]: Prettier auto format

File: .github/workflows/lighthouse.yml [hunk 2]
```diff
@@ -100,83 +99,83 @@ jobs:
...
-                          },
-                          {
-                            "type": "divider"
-                          },
-                          {
-                            "type": "section",
-                            "text": {
-                              "type": "mrkdwn",
-                              "text": "*User: <${{ github.event.issue.user.url }}|${{ github.event.issue.user.login }}>*\n*Link:* ${{ github.event.issue.pull_request.html_url }}\n*Title:* ${{ github.event.issue.title }}\n*Status:* ${{ github.event.issue.state }}"
+                      {
```
Comment (balakrishna-deriv) [inline .github/workflows/lighthouse.yml:102]: Prettier auto format

File: .github/workflows/smoketests.yml [hunk 3]
```diff
@@ -8,68 +8,68 @@ permissions:
...
-        message: "${{ steps.set_msg.outputs.msg }}"
-        recreate: true
+            - name: Leave comment
+              if: always()
+              uses: marocchino/sticky-pull-request-comment@v2
+              with:
+                  header: Smoke tests status update
+                  number: ${{ github.event.issue.number }}
+                  message: '${{ steps.set_msg.outputs.msg }}'
+                  recreate: true
```
Comment (balakrishna-deriv) [inline .github/workflows/smoketests.yml:75]: Prettier auto format

File: .github/workflows/lighthouse.yml [hunk 4]
```diff
@@ -26,9 +26,9 @@ jobs:
                   header: lighthouse
                   message: |
                       Running Lighthouse audit...
-            - name: Capture Vercel preview URL
-              id: vercel_preview_url
-              uses: binary-com/vercel-preview-url-action@v1.0.5
+            - name: Capture preview URL
+              id: capture_preview_url
+              uses: deriv-com/capture-url-from-issue-comment@v1.0.4
```
Comment (balakrishna-deriv) [inline .github/workflows/lighthouse.yml:31]: A new github action to capture CF pages preview link

File: .github/workflows/lighthouse.yml [hunk 5]
```diff
@@ -100,83 +99,83 @@ jobs:
...
+                              "type": "divider"
                             }
-                          },
-                          {
-                            "type": "divider"
-                          }
-                        ]
-                      }
+                          ]
+                        }
```
Comment (balakrishna-deriv) [inline .github/workflows/lighthouse.yml:179]: Auto format

File: .github/workflows/smoketests.yml [hunk 6]
```diff
@@ -8,68 +8,68 @@ permissions:
...
+            fail-fast: false
+            matrix:
+                containers: [1, 2, 3, 4, 5]
+
+        steps:
+            - name: Capture preview URL
+              id: capture_preview_url
+              uses: deriv-com/capture-url-from-issue-comment@v1.0.4
+              with:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (balakrishna-deriv) [inline .github/workflows/smoketests.yml:28]: A new github action to capture CF pages preview link

**Review by adrienne-deriv — APPROVED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**